### ISSUE TITLE
Локација гласачког места - мени за избор

### ISSUE THREAD

**Issue by icemarkom** [lvazic/glasanje#3]
У случају да је позната информација где се жели алтернативно место гласања које није амбасада, можда не би било лоше понудити избор уместо поља за слободан унос података.

Нпр. за Сан Франциско и околину, конзулат препоручује да се унесе *"Сан Франциско / Сан Хозе"* као локација. Или нпр. за Исланд *"Рејкјавик"*.

**Comment by lvazic** [comment]
Ако добро разумем, идеја је:
Понудити пар познатих ставки али понудити и слободан унос текста.

Није тешко додати али моја брига је:

За залив Сан Франциска знамо шта да унесемо јер смо ми звали конзулат и питали. Како да знамо ту информацију за остала места? И како да генерализујемо?

Да ли можда да једноставно препоручимо да унесу најближи велики град или престоницу државе?

**Comment by icemarkom** [comment]
Да, мислим да си добро разумео. Још боље би било ако би могло да се препожна на основу адресе, али то може да буде компликовано.

За места за која се зна, мислим да је ОК да се ставе као понуђено. Како се сазна за ново место, може да се дода. Инструкција за најближи град: СГТМ.

**Comment by lvazic** [comment]
Било шта што укључује геолокацију (АПИ позиви за адресу, захтев са локацију корисника) је нешто што не желимо да имамо, приватност је број 1.

@vokativ шта ти мислиш о предложеној имплементацији? По мом мишљењу требало би да додамо или листу знаних правила или предлог за унос најближег великог града али не оба.

**Comment by icemarkom** [comment]
Ја мислим да може да се направи са обе варијанте. 

```
<select name="alternativnalokacija>
<option selected>Слободан унос
<option>Сан Франциско
<option>Неки град
</select>

<input...>
```

Поље за слободан унос може да се сакрива/открива на основу тога која је ставка изнад изабрана. Пропратни текст може да појасни шта се очекује и шта су конзулати/амбасаде препоручиле.

**Comment by lvazic** [comment]
Додавање листе знаних правила као избора уместо као упутства испод поља за унос може да ради само ако је листа знана и коначна.
У супротном један део корисника добија смислен предлог а други се пита зашто нема ничега близу њих.

То је макар моје мишљење, хајде да чујемо шта други имају да кажу :)

**Comment by vokativ** [comment]
Марко, хвала пуно на цимању.
Да, у принципу, било би много боље да понудимо опције људима да се не распрше са захтевима.
Искрено, нисмо имали времена да се усрередимо на тај део тако да не занм да ли ћемо се разумети до 25ог, али ово тотално има смисла да решимо.

Искрено, пошто немамо времена да направимо ту листу са предлозима по земљи (то би био један фин мањи пројекат са доста користи за ово и друге могуће иницијативе), удобније би било да некако можемо да делимо линк ка сајту са већ попуњеним местом гласања. Јер има неко идеје како то можемо да урадимо?

Листа је добра идеја, али не знам како ћемо је направити на време за ове изборе за све земље. Иначе, амабасаде и конзулати само кажу "дођите овде" док не споменеш закон о избору народних посланика и да можемо да ставимо шта желимо :-)

Баш ми је драго да неко гледа код. Кажи шта мислиш.

**Comment by vokativ** [comment]
Алтернативно, можда би могли да питамо људе када заврше, дали желе да сачувамо њихову локацију за грађење будућих листа. Шта мислите о томе?
